library/posture.c

```diff
@@ -905,29 +905,46 @@ static bool check_running(uv_loop_t *loop, const char *path) {
 #if _WIN32
     HANDLE sh = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0);
     if (sh == INVALID_HANDLE_VALUE) {
-        ZITI_LOG(ERROR, "failed to get process list: %d", GetLastError());
+        ZITI_LOG(ERROR, "failed to get process list: %lu", GetLastError());
         return result;
     }
-    PROCESSENTRY32W entry = {
-            .dwSize = sizeof(PROCESSENTRY32W)
-    };
+
+    // Set the size of the structure before using it.
+    PROCESSENTRY32 pe32;
+    pe32.dwSize = sizeof( PROCESSENTRY32 );
```

Comment:
nit/style: `sizeof(pe32)` is always correct, even if the variable's type is changed in the future.

Reply:
fixed. thx. i straight up "forklifted" the code, but it makes a lot of sense :)